src/mainboard/clevo/adl-p/acpi/mainboard.asl

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#if CONFIG(DRIVERS_GFX_NVIDIA)
+#include <variant/gpio.h>
+#endif
+
+#define EC_GPE_SCI 0x6E
+#define EC_GPE_SWI 0x6B
+#include <ec/system76/ec/acpi/ec.asl>
+
+Scope (\_SB) {
+	#include "sleep.asl"
+	Scope (PCI0) {
+		#include "backlight.asl"
+
+#if CONFIG(DRIVERS_GFX_NVIDIA)
```

Comment:
We don't have this driver, remove. And NS70PU does not have a dGPU

Reply:
Removed the parts in `CONFIG(DRIVERS_GFX_NVIDIA)`.